storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Test.Example/SmokeTests.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using NUnit.Framework;
+using Dotnet.Storm.Example;
+using Dotnet.Storm.Adapter.Test;
+using Dotnet.Storm.Adapter.Components;
+using System.Collections.Generic;
+
+namespace Dotnet.Storm.Test.Example
+{
+    public class SmokeTests
+    {
+        [Test]
+        public void TestCacheChannel()
+        {
+            // Initialize configuration
+            Dictionary<string, object> config = new Dictionary<string, object>();
+            config["configkey1"] = "configvalue1";
+            config["configkey2"] = "configvalue2";
+
+            // Initialize context
+            StormContext sc = new StormContext();
+            sc.ComponentId = "componentid1";
+            sc.StreamToOputputFields = new Dictionary<string, List<string>>();
```

Comment:
Encapsulate this - 
something like sc.SetOutputFields("default", new string[] { "default" })